docs/source/using/managing-minishift.adoc

```diff
@@ -292,3 +292,19 @@ $ minishift ssh -- docker ps
 CONTAINER    IMAGE                   COMMAND                CREATED        STATUS        NAMES
 71fe8ff16548 openshift/origin:v1.5.1 "/usr/bin/openshift s" 4 minutes ago  Up 4 minutes  origin
 ----
+
+
+[[experimental-functionality]]
+== Experimental functionality
+
+To allow users and developers to get early access to some upcoming features, we offer a way to enable experimental functionality.
+
```

Comment:
@gbraad I came up following as I was thinking in this line about the content. WDYT? you can change the content as you like.  cc @thatdocslady @hferentschik 

--------------------------
**Topic: Using new experimental features of `cluster up`**

Minishift uses `oc cluster up` to provision the OpenShift instance in the VM. When `cluster up` introduces a new flag, it used to need code changes to expose it through Minishift CLI and then an user can use the feature in Minishift. But exposing the flags through Minishift takes some time as Minishift project has its own release schedule. So to allow users and developers to get early access to upcoming `cluster up` experimental features, we have come up with this feature. 

For this you need to set an environment variable: